collectors/ebpf.plugin/ebpf_socket.c

```diff
@@ -289,30 +308,23 @@ static void ebpf_socket_send_data(ebpf_module_t *em)
 
     // We read bytes from function arguments, but bandiwdth is given in bits,
     // so we need to multiply by 8 to convert for the final value.
-    write_count_chart(
-      NETDATA_TCP_FUNCTION_COUNT, NETDATA_EBPF_FAMILY, socket_publish_aggregated, 3);
-    write_io_chart(
-        NETDATA_TCP_FUNCTION_BITS, NETDATA_EBPF_FAMILY, socket_id_names[0], common_tcp.write*8/1000,
-        socket_id_names[1], common_tcp.read*8/1000);
+    write_count_chart(NETDATA_TCP_FUNCTION_COUNT, NETDATA_EBPF_IP_FAMILY, socket_publish_aggregated, 3);
+    write_io_chart(NETDATA_TCP_FUNCTION_BITS, NETDATA_EBPF_IP_FAMILY, socket_id_names[0],
+                   common_tcp.write*8/1000, socket_id_names[1], common_tcp.read*8/1000);
     if (em->mode < MODE_ENTRY) {
-        write_err_chart(
-          NETDATA_TCP_FUNCTION_ERROR, NETDATA_EBPF_FAMILY, socket_publish_aggregated, 2);
-    }
-    write_count_chart(
-        NETDATA_TCP_RETRANSMIT, NETDATA_EBPF_FAMILY, &socket_publish_aggregated[NETDATA_IDX_TCP_RETRANSMIT],
-        1);
-
-    write_count_chart(
-        NETDATA_UDP_FUNCTION_COUNT, NETDATA_EBPF_FAMILY, &socket_publish_aggregated[NETDATA_IDX_UDP_RECVBUF],
-        2);
-    write_io_chart(
-        NETDATA_UDP_FUNCTION_BITS, NETDATA_EBPF_FAMILY,
-        socket_id_names[3],(long long)common_udp.write*8/100,
-        socket_id_names[4], (long long)common_udp.read*8/1000);
+        write_err_chart(NETDATA_TCP_FUNCTION_ERROR, NETDATA_EBPF_IP_FAMILY, socket_publish_aggregated, 2);
+    }
+    write_count_chart(NETDATA_TCP_RETRANSMIT, NETDATA_EBPF_IP_FAMILY,
+                      &socket_publish_aggregated[NETDATA_IDX_TCP_RETRANSMIT],1);
+
+    write_count_chart(NETDATA_UDP_FUNCTION_COUNT, NETDATA_EBPF_IP_FAMILY,
+                      &socket_publish_aggregated[NETDATA_IDX_UDP_RECVBUF],2);
+    write_io_chart(NETDATA_UDP_FUNCTION_BITS, NETDATA_EBPF_IP_FAMILY,
+                   socket_id_names[3],(long long)common_udp.write*8/100,
+                   socket_id_names[4], (long long)common_udp.read*8/1000);
```

Comment:
```suggestion
                   socket_id_names[3],(long long)common_udp.write*8/1000,
                   socket_id_names[4], (long long)common_udp.read*8/1000);
```

looks like a problem

Reply:
I see we have https://github.com/netdata/netdata/blob/b1ce4fa3b6b8c423716b4ee6cbb56d12a438d737/libnetdata/libnetdata.h#L295

Reply:
Thank you @ilyam8 I am using this preprocessor now.

Reply:
See https://github.com/thiagoftsm/netdata/blob/ada774aa4f43d180e66c2463cccbad4f8ab435ce/collectors/ebpf.plugin/ebpf_socket.c#L312-L313

Reply:
Done, thanks!